cmdline-pull/pubsub_sample.py

```diff
@@ -105,6 +108,30 @@ def create_topic(client, args):
     print 'Topic {} was created.'.format(topic['name'])
 
 
+def grant_gmail_push(client, args):
+    """Grants gmail push notification on this topic."""
+    topic = get_full_topic_name(args.project_name, args.topic)
```

Comment:
I see. This is because we do the following in main:

```python
args.func(client, args)
```
and some functions have 'topic' but 'subscription', and vice versa.
In other words, it's like a common interface among all the sub commands. I'm open to any suggestions though.